packages/clarity-js/src/layout/region.ts

```diff
@@ -1,40 +1,27 @@
-import { Event, Metric, BooleanFlag } from "@clarity-types/data";
+import { Event, BooleanFlag } from "@clarity-types/data";
 import { Box, RegionData } from "@clarity-types/layout";
-import config from "@src/core/config";
-import * as task from "@src/core/task";
-import { clearTimeout, setTimeout } from "@src/core/timeout";
 import encode from "@src/layout/encode";
 import * as dom from "./dom";
 
-let bm: {[key: number]: RegionData} = {};
+let bm: { [key: number]: RegionData } = {};
 let updateMap: number[] = [];
-let timeout: number = null;
 
 export function compute(): void {
-    clearTimeout(timeout);
-    timeout = setTimeout(schedule, config.lookahead);
-}
-
-function schedule(): void {
-    task.schedule(region);
+    // Making region compute synchronous for tracking regions 
+    // TODO: For modern browsers use resizeobserver 
+    region();
 }
 
 async function region(): Promise<void> {
-    let timer = Metric.LayoutCost;
-    task.start(timer);
     let values = dom.regions();
     let viewport = getViewport();
 
     for (let value of values) {
-        if (task.shouldYield(timer)) {
-            await task.suspend(timer);
-            viewport = getViewport();
-        }
+        viewport = getViewport();
```

Comment:
This is not needed if it's sync. We can use the value above. It was only needed it if was async.